StressTests/StressTests.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+        <IsTestProject>true</IsTestProject>
+	    <DefineConstants>$(DefineConstants);JETBRAINS_ANNOTATIONS;CONTRACTS_FULL</DefineConstants>
+	    <DebugType>portable</DebugType>
+	    <ImplicitUsings>enable</ImplicitUsings>
+	    <NoLogo>true</NoLogo>
+	    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+        
+    </PropertyGroup>
+	<PropertyGroup Condition="'$(Configuration)'=='Debug'">
+		<DefineDebug>true</DefineDebug>
+		<DefineConstants>$(DefineConstants);DEBUG_IDISPOSABLE</DefineConstants>
+	</PropertyGroup>
+	<PropertyGroup Condition="'$(Configuration)'=='Release'">
+		<Optimize>true</Optimize>
+	</PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="coverlet.collector" Version="6.0.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+        <PackageReference Include="xunit" Version="2.5.3" />
+        <PackageReference Include="xunit.runner.visualstudio" Version="2.5.3" />
+    </ItemGroup>
```

Comment:
Should be used the same nuget packages versions as the `UnitTests` project?

Reply:
Hey Grok, for c#, is there a way of having multiple projects (.csproj) that are part of a .sln, share PackageReferences such that all projects are always using the same versions of dependent libraries?

> Yes, in C#, there is a way to ensure that multiple projects (.csproj) within a solution (.sln) share PackageReferences such that all projects consistently use the same versions of dependent libraries. This can be achieved using central package management with a Directory.Packages.props file, a feature supported in the .NET SDK (starting with .NET 5 and improved in later versions).

tee-hee. I love AI.